packages/fx-core/src/component/generator/error.ts

```diff
@@ -50,29 +51,48 @@ export class UnzipError extends BaseComponentInnerError {
 }
 
 export class DownloadSampleNetworkError extends BaseComponentInnerError {
-  constructor(url: string) {
+  constructor(url: string, error: Error) {
     super(
       errorSource,
       "UserError",
       "DownloadSampleNetworkError",
       "error.generator.DownloadSampleNetworkError",
-      [url]
+      [url],
+      undefined,
+      undefined,
+      undefined,
+      error
     );
   }
 }
 
 export class DownloadSampleApiLimitError extends BaseComponentInnerError {
-  constructor(url: string) {
+  constructor(url: string, error: Error) {
     super(
       errorSource,
       "UserError",
       "DownloadSampleApiLimitError",
       "error.generator.DownloadSampleApiLimitError",
-      [url]
+      [url],
+      undefined,
+      undefined,
+      undefined,
+      error
```

Comment:
Have you verified that the inner error is included in telemetry?

Reply:
The inner error is included but only it's message, stack info are included, I will see how to include other properties.